registry/api/errcode/errors_test.go

```diff
@@ -109,28 +108,6 @@ func TestErrorsManagement(t *testing.T) {
 		t.Fatalf("unexpected json:\ngot:\n%q\n\nexpected:\n%q", string(p), expectedJSON)
 	}
 
-	// Now test the reverse
```

Comment:
You can't just delete these tests. These round trips are part of the behavior of the error type.